src/components/LandPage/TableRow/TableRow.tsx

```diff
@@ -21,21 +22,29 @@ const sortLandPoolLast = (a: string, b: string) => {
 
 export default class TableRow extends React.PureComponent<Props> {
   render() {
-    const { land, deployments, onNavigate } = this.props
-    const coords = getCoords(land)
+    const { land, deployments, rental, onNavigate } = this.props
+    const { x, y } = getCoords(land)
+
     return (
       <Table.Row className="TableRow" onClick={() => onNavigate(locations.landDetail(land.id))}>
         <Table.Cell>
           <Row>
             <Column width={67} grow={false} shrink={false}>
               <Atlas landId={land.id} width={45} height={45} isDraggable={false} size={9} />
             </Column>
-            <Column className="name">{land.name}</Column>
+            <Column className="name">
+              <div>
```

Comment:
Is this div redundant?

Reply:
No! it's because Column has a `display: flex` with `flex-direction: column`. Instead of messing with css and overriding rules, I just added a div and it works correctly